<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NewCity</class>
 <widget class="QMainWindow" name="NewCity">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>271</width>
    <height>132</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>271</width>
    <height>132</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>Arial</family>
    <pointsize>12</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Legg til nytt sted</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>jobber.png</normaloff>jobber.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QFormLayout" name="formLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="labelCityName">
      <property name="locale">
       <locale language="NorwegianBokmal" country="Norway"/>
      </property>
      <property name="text">
       <string>Stedsnavn</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="lineEditCityName"/>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="labelCountryID">
      <property name="locale">
       <locale language="NorwegianBokmal" country="Norway"/>
      </property>
      <property name="text">
       <string>Landnummer</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QComboBox" name="comboBox"/>
    </item>
    <item row="3" column="1">
     <widget class="QLabel" name="labelCountryName">
      <property name="locale">
       <locale language="NorwegianBokmal" country="Norway"/>
      </property>
      <property name="text">
       <string>$land</string>
      </property>
     </widget>
    </item>
    <item row="4" column="0" colspan="2">
     <widget class="QDialogButtonBox" name="buttonBox">
      <property name="locale">
       <locale language="NorwegianBokmal" country="Norway"/>
      </property>
      <property name="standardButtons">
       <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
